client/apps/edit/components/layout/index.coffee

```diff
@@ -188,9 +188,15 @@ module.exports = class EditLayout extends Backbone.View
     @fullText.push('<img></img>') for num in [imageCount..1]
     @fullText = @fullText.join(' ')
     
+    @yoastTitle = ''
+    if @article.get('thumbnail_title')
+      @yoastTitle = @article.get('thumbnail_title')
+    else
+      @yoastTitle = @article.get('title')
+
     yoastView = new YoastView
       contentField: @fullText
-      title: @article.get('title')
+      title: @yoastTitle
```

Comment:
ooh we can make this a one-line PR, ready? 
```
title: @article.get('thumbnail_title') or @article.get('title') or ''
```
and then if possible, we should add a test! you should be able to see what gets passed into YoastView by doing something after [this line](https://github.com/artsy/positron/blob/master/client/apps/edit/components/layout/test/index.coffee#L152)
```
console.log @EditLayout.YoastView.args
```